google/cloud/storage/emulator/emulator.py

```diff
@@ -99,6 +149,7 @@ def root_put_object_with_bucket(bucket_name, object_name):
 
 
 @gcs.route("/b", methods=["GET"])
+@retry_test(method="storage.buckets.list")
```

Comment:
Okay, I think I understand how this works, you can wrap any other API with the retry test wrapper, with minimal effort.  Very neat. Still does not answer why we need a "retry test" resource...